bn.tex

```diff
@@ -0,0 +1,1913 @@
+\documentclass[synpaper]{book}
```

Comment:
Still here...

Reply:
that file is in a base repo libtom/libtommath:master